cypress/e2e/Mobile_Check.js

```diff
@@ -0,0 +1,42 @@
+// Mobile_Check.js
+// This test aims to test what a user will encounter if they
+// attempt to complete the experiment on a mobile device
+
+describe("normal_paths", () => {
```

Comment:
the name of the test set wants to be changed from "normal_paths" to "mobile_check"